packages/typo3-docs-theme/resources/template/structure/layout.html.twig

```diff
@@ -160,7 +160,6 @@ j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
 <script src="{{ getRelativePath('_resources/js/popper.min.js') }}"></script>
 <script src="{{ getRelativePath('_resources/js/bootstrap.min.js') }}"></script>
 <script src="{{ getRelativePath('_resources/js/theme.min.js') }}"></script>
-<script src="{{ getRelativePath('_resources/js/code-clipboard.min.js') }}"></script>
```

Comment:
We could also include popper and bootstrap into the minified file, so that we only need to load one file?

Reply:
Yup, that is another story. But I an not really sure, if we really need popper.